data/deprecated.json

```diff
@@ -1253,6 +1253,138 @@
     "old": {"parking": "street"},
     "replace": {"parking": "lane"}
   },
+  {
+    "old": {"parking:lane:both": "yes"},
+    "replace": {"parking:both": "yes"}
+  },
+  {
+    "old": {"parking:lane:both": "no"},
+    "replace": {"parking:both": "no"}
+  },
+  {
+    "old": {"parking:lane:both": "no_parking"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_parking"}
+  },
+  {
+    "old": {"parking:lane:both": "no_standing"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_standing"}
+  },
+  {
+    "old": {"parking:lane:both": "no_stopping"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_stopping"}
+  },
+  {
+    "old": {"parking:lane:both": "fire_lane"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_stopping", "parking:both:restriction:reason": "fire_lane"}
+  },
```

Comment:
🟡  671  https://taginfo.openstreetmap.org/tags/parking%3Alane%3Aboth=fire_lane

Reply:
I think we can keep this (even though below 1k) because the new tagging is complex.